src/pair.py

```diff
@@ -3,3 +3,25 @@
 # Part of azplugins, released under the BSD 3-Clause License.
 
 """Pair potentials."""
+
+from hoomd.azplugins import _azplugins
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.md import pair
+
+
+class PerturbedLennardJones(pair.Pair):
```

Comment:
Add your potential by adapting this class. Classes should usually be added alphabetically.